client/js/table-view.js

```diff
@@ -78,15 +80,43 @@ class TableView {
 		this.sheetBodyEl.appendChild(fragment);
 	}
 
+	renderTableFooter() {
```

Comment:
My apologizes, I set my Sublime Text preferences but I guess I didn't set it for the files.